v1/src/main/java/com/google/cloud/teleport/spanner/ddl/InformationSchemaScanner.java

```diff
@@ -445,7 +442,7 @@ Statement listIndexesSQL() {
       case GOOGLE_STANDARD_SQL:
         return Statement.of(
             "SELECT t.table_schema, t.table_name, t.index_name, t.parent_table_name, t.is_unique,"
-                + " t.is_null_filtered, t.index_type, t.search_partition_by, t.search_order_by"
+                + " t.is_null_filtered, t.filter, t.index_type, t.search_partition_by, t.search_order_by"
```

Comment:
I think this may be causing test failures - e.g. https://pantheon.corp.google.com/dataflow/jobs/us-central1/2024-09-24_01_47_34-4743833279596043378;bottomTab=JOB_LOGS;expandBottomPanel=true;logsSeverity=ERROR;graphView=0?project=cloud-teleport-testing&pageState=(%22dfTime%22:(%22l%22:%22dfJobMaxTime%22))&e=13802955&mods=-ai_platform_fake_service,ai_platform_staging_service